src/agents/agent.py

```diff
@@ -315,10 +312,12 @@ def execute(self, prompt: str, project_name_from_user: str = None) -> str:
                 project_name,
                 f"I am browsing the web to research the following queries: {queries_combined}."
                 f"\n If I need anything, I will make sure to ask you."
+                f"\n I hope i will ask question about things if it gets confusing"
             )
+
         if not queries and len(queries) == 0:
             self.project_manager.add_message_from_devika(project_name,
-                                                         "I think I can proceed without searching the web.")
+                                                         "I am proceeding searching the web for the best results")
```

Comment:
Why are you writing "I am proceeding searching the web for the best results" when we really do not need any web search if we have no queries?

Reply:
fixed!